### DISCUSSION TITLE
RBM Terminology Review

### DISCUSSION THREAD

**Discussion by jwildfire** [Gilead-Public/gsm#398] [General]
@Gilead-BioStats/gsm - I've been a bit lost in RBM (Risk-based monitoring) alphabet soup over the last few weeks, so I thought I'd do some reading to try to improve my understanding of the various terms and processes. This [link](http://www.transceleratebiopharmainc.com/wp-content/uploads/2017/09/Risk-Based-Quality-Managment.pdf) in particular has helped me to better understand  the RBM terminology relevant to GSM. Please give it a quick read as you have time. 

GSM is (at least for now) squarely focused on the site-level KRI (Key Risk Indicator)/Threshold side of things, though there is certainly the possibility that we'll venture into study-level Parameter/QTL (Quality Tolerance Limits) evaluation eventually. 
For now, I think the terminology we use in GSM is pretty good (Yay!): We perform *risk assessments​* (with configurable *thresholds​*) to generate site-level *KRIs​* at a given time point.

**Comment by jwildfire** [comment]
[This article](https://www.magiworld.org/Journal/2014/1411_Centralized.pdf
) also draws a distinction between CSM (Centralized Statistical Monitoring) and KRI evaluation. It's a little old, but I *think* the terminology is current. Again - we fall in the KRI camp.

**Comment by samussiah** [comment]
Added these links to the [GSM Background wiki](https://github.com/Gilead-BioStats/gsm/wiki/GSM-Background).
